skeleton/docker/Dockerfile

```diff
@@ -64,6 +64,11 @@ COPY *.* ./
 
 # Copy source files
 COPY src ./src
+// @if feat['scaffold-navigation']
+
+# Copy @fortawesome files
+COPY @fortawesome ./@fortawesome
+// @endif
```

Comment:
@zewa666 

I am afraid we don't need [these lines](https://github.com/aurelia/cli/pull/1174/commits/bc595e4974c3621350e9a2dc303e8b75f69ffce7#diff-e37943b4c897a421a4c47b666eaff496R67-R71) here, only the one you've added below on the production-stage will be sufficient. 
The reason is that the `@fortawesome` folder will be added when in the `publish-stage` the `au build --env prod` command is running, so it will be available from then on. 

At this point, the specified folder does not exist resulting in failure of docker commands. check the failed CircleCI tests.

Reply:
I'm not next to the PC at the moment but will change once back. Thx for the review